### PR TITLE
[leodict] new connector

### PR DESCRIPTION
**Pull request by bw1**
with Mojo

### CODE REVIEW

File: scripts/leodict.pl [hunk 1]
```diff
@@ -4,27 +4,45 @@
...
-    authors     => 'Stefan \'tommie\' Tomanek',
-    contact     => 'stefan@pico.ruhr.de',
+    authors     => 'Stefan \'tommie\' Tomanek, bw1',
+    contact     => 'bw1@aol.at',
     name        => 'leodict',
     description => 'translates via dict.leo.org',
     license     => 'GPLv2',
     url         => 'http://irssi.org/scripts/',
-    changed     => $VERSION,
     modules     => 'LWP::Simple Data::Dumper',
```
Comment (vague666) [inline scripts/leodict.pl]: Mojo::DOM missing, but if you are going to use Mojo then using the whole mojo suite is just some steps away, use Mojo::UserAgent to fetch the data, say $ua->get('www.perl.org')->result->dom->at('title')->text; result->json gives json output also, not so many dependencies with just cpanm Mojolicious :)
* it's only suggestions so do what you prefer and feel comfortable with :) *

**Comment by ailin-nemui** [comment]
I think you can change from LWP::Simple to Mojo::UserAgent `get'

**Comment by bw1** [comment]
done

File: scripts/leodict.pl [hunk 2]
```diff
@@ -4,27 +4,45 @@
...
-    contact     => 'stefan@pico.ruhr.de',
+    authors     => 'Stefan \'tommie\' Tomanek, bw1',
+    contact     => 'bw1@aol.at',
     name        => 'leodict',
     description => 'translates via dict.leo.org',
     license     => 'GPLv2',
     url         => 'http://irssi.org/scripts/',
-    changed     => $VERSION,
-    modules     => 'LWP::Simple Data::Dumper',
+    modules     => 'LWP::Simple Encode JSON::PP Mojo::DOM Getopt::Long POSIX',
```
Comment (ailin-nemui) [inline scripts/leodict.pl]: now we can remove lwp

**Comment by bw1** [comment]
done

**Comment by vague666** [comment]
Also, JSON::PP shouldn't be needed since Mojo does json with Mojo::JSON :)

**Pull request merged**